packages/app-explorer/src/BlockHeader/Extrinsics.tsx

```diff
@@ -13,14 +13,14 @@ import translate from '../translate';
 
 type Props = I18nProps & {
   hash: Uint8Array,
-  chainGetBlock: any
+  getBlock: any
```

Comment:
can this `getBlock` here be of type `SignedBlock` from @polkadot/types like it is in BlockByHash.tsx